src/elements/content-explorer/ContentExplorer.js

```diff
@@ -1393,33 +1445,37 @@ class ContentExplorer extends Component<Props, State> {
             <Internationalize language={language} messages={messages}>
                 <div id={this.id} className={styleClassName} ref={measureRef} data-testid="content-explorer">
                     <div className="be-app-element" onKeyDown={this.onKeyDown} tabIndex={0}>
-                        <Header
-                            view={view}
-                            isSmall={isSmall}
-                            searchQuery={searchQuery}
-                            logoUrl={logoUrl}
-                            onSearch={this.search}
-                        />
-                        <SubHeader
-                            view={view}
-                            viewMode={viewMode}
-                            rootId={rootFolderId}
-                            isSmall={isSmall}
-                            rootName={rootName}
-                            currentCollection={currentCollection}
-                            canUpload={allowUpload}
-                            canCreateNewFolder={allowCreate}
-                            gridColumnCount={gridColumnCount}
-                            gridMaxColumns={GRID_VIEW_MAX_COLUMNS}
-                            gridMinColumns={GRID_VIEW_MIN_COLUMNS}
-                            maxGridColumnCountForWidth={maxGridColumnCount}
-                            onUpload={this.upload}
-                            onCreate={this.createFolder}
-                            onGridViewSliderChange={this.onGridViewSliderChange}
-                            onItemClick={this.fetchFolder}
-                            onSortChange={this.sort}
-                            onViewModeChange={this.changeViewMode}
-                        />
+                        {hasHeader && (
```

Comment:
Can you wrap the `Header` and `SubHeader` in a React Fragment (`<>`), so that the `hasHeader` check occurs only once?